.pre-commit-config.yaml

```diff
@@ -46,7 +47,7 @@ repos:
         types: [python]
 
   - repo: https://github.com/PyCQA/bandit
-    rev: "1.7.4" # frozen: 1.7.4
+    rev: "1.7.5"
```

Comment:
Since all other versions are frozen, we should probably also freeze this version?

Reply:
SWP unfreezes it because the poetry lock file doesn't provide a commit afaik.